<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emunand</class>
 <widget class="QDialog" name="Emunand">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>492</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>emuNAND creation Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/emunand.png</normaloff>:/images/emunand.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="emunandTypeBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>124</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Select emuNAND type:</string>
   </property>
   <widget class="QRadioButton" name="emunandType_PartitionChkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SD Partition</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="emunandType_SDFileAMSChkBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SD File (AMS)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="emunandType_SDFileSXChkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SD File (SX OS)</string>
    </property>
   </widget>
   <widget class="QLabel" name="emunandType_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>321</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="bootPartBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Input NAND:</string>
   </property>
   <widget class="QLineEdit" name="boot0_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>52</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="boot0_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>BOOT0:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boo0_pushBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>52</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boo1_pushBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>82</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file...</string>
    </property>
   </widget>
   <widget class="QLabel" name="boot1_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>82</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>BOOT1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="boot1_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>82</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nand_pushBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>22</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nand_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>22</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nand_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NAND:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>466</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createEmunandBtn">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>466</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Create emuNAND</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="outputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>368</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Output:</string>
   </property>
   <widget class="QProgressBar" name="outBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>0</height>
     </size>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="outPix1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(124, 208, 170);</string>
    </property>
   </widget>
   <widget class="QLabel" name="outLbl1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="outLbl2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="outPix2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 70, 164);</string>
    </property>
   </widget>
   <widget class="QLabel" name="outPathTitleLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Path: </string>
    </property>
   </widget>
   <widget class="QLabel" name="outPathLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="driveListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>234</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Select target:</string>
   </property>
   <widget class="QListWidget" name="driveList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
